src/index.ts

```diff
@@ -94,7 +94,7 @@ const flatten = <T>(arr: T[][]): T[] => {
  * @param {LookPathOption} opt Options for lookpath.
  * @return {Promise<string>} Resolves absolute file path, or undefined if not found.
```

Comment:
```suggestion
 * @return {Promise<string|undefined>} Resolves absolute file path, or undefined if not found.
```

Reply:
Unlike JSDoc, TSDoc in TypeScript does not require type annotation as TypeScript code already have one. But this is your code, your choice.